iap_alternative_provider/models/iap_account.py

```diff
@@ -9,7 +9,6 @@ class IapAccount(models.Model):
     _inherit = "iap.account"
     _name = "iap.account"
```

Comment:
This line is not needed

Reply:
I don't know the module, but maybe there is a reason it combines both `_inherit` and `_name`, cf. https://www.odoo.com/documentation/17.0/developer/reference/backend/orm.html#classical-inheritance

Also, this line is present since 12.0

Reply:
please clarify if you want it kept or removed so we can move forward with the merge.

Reply:
It is not needed, the model already exists and you need to define the name if you want to create a new module or have multiple inheritances. It is not the case